code/datums/components/edible.dm

```diff
@@ -0,0 +1,270 @@
+/*!
+
+This component makes it possible to make things edible. What this means is that you can take a bite or force someone to take a bite (in the case of items).
+These items take a specific time to eat, and can do most of the things our original food items could.
+
+Behavior that's still missing from this component that original food items had that should either be put into seperate components or somewhere else:
+	Components:
+	Drying component (jerky etc)
+	Customizable component (custom pizzas etc)
+	Processable component (Slicing and cooking behavior essentialy, making it go from item A to B when conditions are met.)
+	Dunkable component (Dunking things into reagent containers to absorb a specific amount of reagents)
+
+	Misc:
+	Something for cakes (You can store things inside)

```

Comment:
This applies to loaves of bread and wheels of cheese as well.

Reply:
ye I'll be sure to go over all edge-case food when I get to the third PR where I'll kill the current food paths